docs/js/online.js

```diff
@@ -30,6 +30,14 @@ $(document).ready(function() {
     return problems;
   });
 
+  function inspect(obj, opts) {
+    if (typeof util === 'object') {
+      return util.inspect(obj, opts);
+    }
+    console.warn("node-util-inspect not supported on this browser")
+    console.log(obj)
+    return String(obj) + "     // See browser console"
```

Comment:
I think it would be better to swap hint and object representation, so the hint is always be visible in case of big objects
```suggestion
    return "// See browser console. Stringifyed result:\n" + String(obj)
```